MvvmCross.Android.Support/V7.AppCompat/MvxAppCompatViewPresenter.cs

```diff
@@ -568,7 +568,7 @@ protected virtual new Fragment GetFragmentByViewType(Type type)
 
         protected virtual Fragment FindFragmentInChildren(string fragmentName, FragmentManager fragManager)
         {
-            if (fragManager?.Fragments != null && !fragManager.Fragments.Any()) return null;
+            if (!(fragManager?.Fragments?.Any() ?? false)) return null;
```

Comment:
This could be done in another PR.

Reply:
yes, can definitely move to different PR.  Need to look at why Playground.Droid is failing to go to MasterDetails page. Don't think it's specific to this PR.